packages/terra-site/src/examples/table/NoStripedTable.jsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import Table from 'terra-table';
+
+const NoStripedTable = () => (
+  <Table isStriped={false}>
+    <Table.Header>
+      <Table.HeaderContent content={'Column Heading 1'} key={1} minWidth={'small'} />
+      <Table.HeaderContent content={'Column Heading 2'} key={2} minWidth={'medium'} />
+      <Table.HeaderContent content={'Column Heading 3'} key={3} minWidth={'large'} />
+    </Table.Header>
+    <Table.Rows>
+      <Table.Row key={0}>
```

Comment:
I know that it's not good practice to set the key equal to the index of the array, and this section of code feels like it's doing just that, even though there isn't a map or forEach function being called. Could we generate unique ids for these keys instead of using the numbers 1, 2, and 3? I used [shortid](https://github.com/dylang/shortid) to generate it in one of my code changes for example.